### DISCUSSION TITLE
How do I get the secret to update again when I update the secret manager side at aws ?

### DISCUSSION THREAD

**Discussion by yongxiuuu** [external-secrets/kubernetes-external-secrets#517] [Q&A]

**Comment by yongxiuuu** [comment]
Currently, I did a few steps to update a new password: 

1) Delete secret
2) Delete externalsecret
3) kubectl apply externalsecret.yaml
4) restart the app deployment and it got updated. 

If anyone has a less tedious please share! Thank you!

**Comment by Flydiverny** [comment]
Step 2 and 3 should be enough from KES perspective. 
You could also modify something in the secret (an annotation with a counter or something), its not ideal.
There's some pending PRs that should discover deletion of secret and recreate and a webhook PR to allow triggering a sync as well :)

For 4 you could take a look at something like https://github.com/stakater/Reloader
